### PR TITLE
[DEV-1094]: Login MFA flow

### PR DESCRIPTION
**Pull request by MikeAtUqido**
#### List of Changes

* Added the second step in the login flow

#### Motivation and Context

The login flow needs an MFA flow

#### How Has This Been Tested?

[x] Manually

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: apps/nextjs-website/src/_contents/translations.ts [hunk 1]
```diff
@@ -332,6 +332,13 @@ export const translations = {
       forgotPassword: 'Hai dimenticato la password?',
       noAccount: 'Non hai un account?',
     },
+    confirmlogin: {
```
Comment (datalek) [inline apps/nextjs-website/src/_contents/translations.ts:335]:
```suggestion
    confirmLogin: {
```

File: apps/nextjs-website/src/_contents/translations.ts [hunk 2]
```diff
@@ -332,6 +332,13 @@ export const translations = {
...
+      title: 'Verifica di accesso MFA',
+      body: 'Inserisci il codice di verifica inviato nella tua e-mail',
+      code: '012345',
+      wrongaccount: 'Hai sbagliato account? ',
```
Comment (datalek) [inline apps/nextjs-website/src/_contents/translations.ts:339]:
```suggestion
      wrongAccount: 'Hai sbagliato account? ',
```

**Review by datalek — CHANGES_REQUESTED** [review]

File: apps/nextjs-website/src/app/auth/login/page.tsx [hunk 3]
```diff
@@ -27,7 +62,11 @@ const Login = () => {
         my={6}
         spacing={6}
       >
-        <LoginForm />
+        {logInStep === LoginSteps.LOG_IN && <LoginForm onLogin={onLogin} />}
+        {logInStep === LoginSteps.MFA_CHALLENGE && (
+          // TODO: Create this component
```
Comment (datalek) [inline apps/nextjs-website/src/app/auth/login/page.tsx:67]:
```suggestion
```

File: apps/nextjs-website/src/components/organisms/Auth/ConfirmLogin.tsx [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+  const {
+    auth: { confirmlogin },
+    shared,
+  } = translations;
+
+  const [error, setError] = useState<string | null>(null);
+  const [code, setCode] = useState<string>('');
+
+  const onConfirmLoginHandler = useCallback(() => {
+    confirmLogin(code).catch((e) => setError(e.message));
```
Comment (datalek) [inline apps/nextjs-website/src/components/organisms/Auth/ConfirmLogin.tsx:34]: The user has only one tentative to send the code. So on failure we should not allow another tentative

File: apps/nextjs-website/src/components/organisms/Auth/ConfirmLogin.tsx [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+            <Divider />
+            <Stack
+              pt={4}
+              pb={8}
+              display='flex'
+              alignItems='center'
+              justifyContent='center'
+              flexDirection='row'
+            >
+              <Typography variant='caption-semibold' mr={1}>
```
Comment (datalek) [inline apps/nextjs-website/src/components/organisms/Auth/ConfirmLogin.tsx:86]: It's useful a "resend" link which is missing

**Review by datalek — CHANGES_REQUESTED** [review]

File: apps/nextjs-website/src/components/organisms/Auth/ConfirmLogin.tsx (hunk 4 above)
Comment (MikeAtUqido) [reply apps/nextjs-website/src/components/organisms/Auth/ConfirmLogin.tsx]: Here's the activity: [DEV-1128](https://pagopa.atlassian.net/browse/DEV-1128)

File: apps/nextjs-website/src/components/organisms/Auth/ConfirmLogin.tsx (hunk 5 above)
Comment (MikeAtUqido) [reply apps/nextjs-website/src/components/organisms/Auth/ConfirmLogin.tsx:86]: Here's the activity: [DEV-1129](https://pagopa.atlassian.net/browse/DEV-1129)

**Review by datalek — APPROVED** [review]

**Review by kin0992 — APPROVED** [review]

**Pull request merged**